### PR TITLE
chore(react-chart): min/max applying for scale processing

### PR DESCRIPTION
**Pull request by Krijovnick**

### CODE REVIEW

File: packages/dx-chart-core/src/plugins/integrated-scale-processing/computeds.js [hunk 1]
```diff
@@ -6,11 +6,10 @@ const isDefined = item => item !== undefined;
 const getAxesDomains = axes =>
```
Comment (alexey-semikozov) [inline packages/dx-chart-core/src/plugins/integrated-scale-processing/computeds.js]: Rename this function OR remove it. Now it does nothing

File: packages/dx-chart-core/src/plugins/integrated-scale-processing/computeds.js [hunk 2]
```diff
@@ -55,7 +54,28 @@ const calculateDomain = (series, data, axesDomains, argumentAxisName) =>
     axesDomains,
   );
 
+const zoomingDomains = (axes, calculatedDomains) => axes.reduce(
```
Comment (alexey-semikozov) [inline packages/dx-chart-core/src/plugins/integrated-scale-processing/computeds.js]: I think "zoomingDomains" is not a good name for this function.

File: packages/dx-chart-core/src/plugins/integrated-scale-processing/computeds.js (hunk 1 above)
Comment (Krijovnick) [reply packages/dx-chart-core/src/plugins/integrated-scale-processing/computeds.js]: What about 'collectAxesTypes' name?

File: packages/dx-chart-core/src/plugins/integrated-scale-processing/computeds.js (hunk 2 above)
Comment (Krijovnick) [reply packages/dx-chart-core/src/plugins/integrated-scale-processing/computeds.js]: Maybe 'adjustDomains'?

File: packages/dx-chart-core/src/plugins/integrated-scale-processing/computeds.js (hunk 1 above)
Comment (alexey-semikozov) [reply packages/dx-chart-core/src/plugins/integrated-scale-processing/computeds.js]: nice 👍

File: packages/dx-chart-core/src/plugins/integrated-scale-processing/computeds.js (hunk 2 above)
Comment (alexey-semikozov) [reply packages/dx-chart-core/src/plugins/integrated-scale-processing/computeds.js]: good

**Pull request merged**